cli/bpmetadata/tfconfig.go

```diff
@@ -235,13 +244,38 @@ func getBlueprintInterfaces(configPath string) (*BlueprintInterface, error) {
 		variables = append(variables, v)
 	}
 
+	p := hclparse.NewParser()
+	variableFile, hclDiags := p.ParseHCLFile(filepath.Join(configPath, "variables.tf"))
+	if hclDiags.HasErrors() {
+		Log.Info("Failed to parse HCL: ", "diags: ", hclDiags)
+	}
+	variableContent, _, hclDiags := variableFile.Body.PartialContent(variableSchema)
```

Comment:
How about calling `log.Error()` instead? Or do you have any suggestions?

Reply:
My recommendation would be to propagate the error from this function (change signature to (map[string]int, error) and only sort if no error, otherwise log and continue so we degrade gracefully. IRL if this throws an error, we would not reach here as it will likely be caught earlier in tfconfig.LoadModule.